pkg/kubelet/dockershim/network/hostport/hostport_manager.go

```diff
@@ -38,8 +38,8 @@ import (
 	utilnet "k8s.io/utils/net"
 )
 
-// HostPortManager is an interface for adding and removing hostport for a given pod sandbox.
-type HostPortManager interface {
+// Manager is an interface for adding and removing hostport for a given pod sandbox.
```

Comment:
Changing this makes the code less readable and I'd be tempted to leave it as is :)